pom.xml

```diff
@@ -613,6 +613,11 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>org.rocksdb</groupId>
+        <artifactId>rocksdbjni</artifactId>
+        <version>5.5.1</version>
```

Comment:
Why not use the latest version of rocksdbjni?
https://search.maven.org/search?q=g:org.rocksdb

Reply:
good call, updated to `5.15.10`